### DISCUSSION TITLE
What does this mean?

### DISCUSSION THREAD

**Discussion by crichmond** [dotnettools/SharpGrabber#70] [General]
Under SharpGrabber.Desktop ...new 4.2 release notes it says:

Grabs from every source supported by official grabbers

What does that mean?

**Comment by javidsho** [comment]
It means that the desktop application puts every package provided in this repository into use. That is, YouTube, Vimeo, Black Widow scripts etc.

**Comment by crichmond** [comment]
Ahh...excellent. Thanks.  When you said "officially supported grabbers" I
thought you were referring to some standard protocol or list of video
vendors.

Thanks.
